test/e2e/apimachinery/resource_quota.go

```diff
@@ -1418,7 +1418,9 @@ func waitForResourceQuota(c clientset.Interface, ns, quotaName string, used v1.R
 		}
 		// verify that the quota shows the expected used resource values
 		for k, v := range used {
-			if actualValue, found := resourceQuota.Status.Used[k]; !found || (actualValue.Cmp(v) != 0) {
+			if actualValue, found := resourceQuota.Status.Used[k]; !found ||
```

Comment:
I a bit worried that this change makes our tests more permissive and could hide some bugs in the future. Could we possibly make changes to the tests or our test initialization so that we can keep != check? For example, tests may want to drop all existing quota. Or, we may want to add them to the expected values.